doc/source/whatsnew/v0.20.0.txt

```diff
@@ -31,7 +31,15 @@ Other enhancements
 ^^^^^^^^^^^^^^^^^^
 
 
+ - The ``dtype`` keyword argument in the :func:`read_csv` function for specifying the types of parsed columns
+ is now supported with the ``'python'`` engine.  See the :ref:`io docs <io.dtypes>` for more information.
 
+.. ipython:: python
+
+   from io import StringIO
```

Comment:
this will not work on python 2. Should check how it is done in other places in the docs